src/console_commands.h

```diff
@@ -35,6 +35,8 @@ class ConsoleCommands : public nocopy_nomove
 	uv_loop_t m_loop;
 	uv_async_t m_shutdownAsync;
 	uv_tty_t m_tty;
+	uv_pipe_t m_stdin_pipe;
+	uv_handle_t* m_stdin_handle;
```

Comment:
`uv_stream_t* m_stdin_handle` because it will point to a stream handle, not just a handle. `uv_pipe_t` and `uv_tty_t` are both stream handles.